src/Tvheadend.cpp

```diff
@@ -2634,6 +2645,28 @@ bool CTvheadend::ParseEvent ( htsmsg_t *msg, bool bAdd, Event &evt )
     evt.SetYear(u32);
   if (!htsmsg_get_u32(msg, "dvrId", &u32))
     evt.SetRecordingId(u32);
+
+  /* Extra checks for HTSP v32+ */
+  if (m_conn->GetProtocol() >= 32) {
+    /* 
+      Due to changes in HTSP v32, try to use the summary
+      as the description. If the summary is empty, try to use
+      the subtitle as the description. Also clear the respective
+      entries to avoid duplicate information being displayed.
+
+      This was done by TVHeadend prior to HTSP v32.
+    */
```

Comment:
This comment block fits best after `if (evt.GetDesc().empty()) {`, not before. Please move the comment.